src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -77,20 +78,20 @@ pub mod shape {
 
             // === Selection ===
 
-            let selection_offset = 4.px();
-            let selection_size   = 7.px();
-            let select_width   = &width  - 2.px() + &selection_offset * 2.0 * &selection;
-            let select_height  = &height - 2.px() + &selection_offset * 2.0 * &selection;
-            let select_radius  = &select_height / 2.0;
-            let select         = Rect((&select_width,&select_height)).corners_radius(&select_radius);
+            let selection_offset = 5.px();
+            let selection_size   = 9.px();
```

Comment:
These two could be a part of theme as well, however it is not obligatory for this PR.